code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -155,6 +155,72 @@
 	else
 		..()
 
+/mob/living/simple_animal/friendly/sheep
+	name = "sheep"
+	desc = "It's so fluffy! can't shear it though. milking is fine."
```

Comment:
I'd adjust the grammar here to match the existing grammar standards, i.e. capitalizing the start of sentences.

Reply:
edited.